src/toml.ads

```diff
@@ -235,6 +235,11 @@ package TOML is
       with Pre => Value.Kind = TOML_Table and then Value.Has (Key);
    --  Likewise, but take an unbounded string
 
+   function Merge (L, R : TOML_Value) return TOML_Value
+      with Pre  => L.Kind = TOML_Table and then R.Kind = TOML_Table,
+           Post => Merge'Result.Kind = TOML_Table;
+   --  Merge two tables. If a key is present in both, Constraint_Error is raised
```

Comment:
As I said in #1, can you please complete this comment to state that elements in the returned table are aliases for L’s and R’s elements (i.e. there is no deep copy).

Also, I wrote it but somehow the message: can you add a new testcase for this new function? Something similar to the one for Clone (https://github.com/pmderodat/ada-toml/tree/master/tests/clone) checking the regular case (expected table content and aliasing) and the error case would be great. Thanks!

Reply:
Ooops, missed that.

Is there any documentation about the e3 test suite? I guess I need https://github.com/AdaCore/e3-testsuite for starters... If I try to run run-tests.py:

```
Traceback (most recent call last):
  File "./run-tests.py", line 16, in <module>
    from e3.fs import sync_tree
ImportError: No module named e3.fs
```

And I can't find e3.fs in the repo...

Reply:
Yes, you need this library, but I guess I need to document how to install it.

The simplest way to do that is to use Python’s own package manager:
```sh
# Create a virtualenv (prefix to install packages)
# The exact command varies from one Linux distrib to another:
# virtualenv, virtualenv2, python2-virtualenv, …
$ virtualenv2 my-virtual-env

# Update your environment to use it
$ source my-virtual-env/bin/activate

# Install e3-testsuite and all its dependencies
$ pip install e3-testsuite

# You should now be able to run the testsuite (make sure you built
# `ada_toml.gpr` and `checkers.adb` first):
$ ./run-tests.py
```

I’ll document this soon :) Sorry for the inconvenience in the meantime!